test/livebook_teams/hubs/team_client_test.exs

```diff
@@ -151,6 +151,47 @@ defmodule Livebook.Hubs.TeamClientTest do
       # we dispatch the `{:event, :deployment_group_updated, :deployment_group}` event
       assert_receive {:deployment_group_updated, %{id: ^id, agent_keys: [^built_in_agent_key]}}
     end
+
+    @tag :tmp_dir
+    test "receives the app events", %{team: team, tmp_dir: tmp_dir} do
+      deployment_group = build(:deployment_group, name: team.id, mode: :online)
+      assert {:ok, id} = Livebook.Teams.create_deployment_group(team, deployment_group)
+
+      id = to_string(id)
+
+      # receives `{:event, :deployment_group_created, :deployment_group}` event
+      assert_receive {:deployment_group_created, %{id: ^id, app_deployments: []}}
+
+      # creates a session to simulate the deploy
+      local = Livebook.FileSystem.Local.new()
+      path = Path.join(tmp_dir, "MyNotebook2.livemd")
+      file = Livebook.FileSystem.File.new(local, path)
```

Comment:
You can do this:

```suggestion
      path = Path.join(tmp_dir, "MyNotebook2.livemd")
      file = Livebook.FileSystem.File.local(path)
```

Let's update in other places too :)

Reply:
Since we're going to not use the `FileSystem.File` abstraction, I'll use the `File` instead

Reply:
Here we are passing the file to the session process, which does expect `FileSystem.File` :D